app/controllers/cart/cart_items_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Cart::CartItemsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    current_cart.add_cart_item(cart_item_params[:product_id], cart_item_params[:quantity])
+    redirect_to cart_url, notice: '商品をカートに追加しました'
+  end
+
+  def update
+    current_cart.cart_items.find(params[:id]).update_quantity!(cart_item_params[:quantity])
```

Comment:
別に
```ruby
before_action :set_cart_item, only: %i[update]

private

def set_cart_item
  @cart_item = current_cart.cart_items.find(params[:id])
end
```

としておいてから

```suggestion
    @cart_item.update_quantity!(cart_item_params[:quantity])
```

とするのが Scaffold に近い状態で Rails っぽい気がしました。

Reply:
`current_cart.update_cart_item(product, cart_item_params[:quantity])` でもよいかも
そうするなら product は set_product を用意して before_action するようにするととても自然な気がしました

Reply:
1つ目の案で修正しました
[9cecfb7](https://github.com/kk0000-kk/sakura-market/pull/25/commits/9cecfb72cc3e18023b583fb830ad4cf9d9c61294)